src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      { age && <p className="Person__age">{`I am ${age}`}</p> }
+      { isMarried
+        ? <p className="Person__partner">{`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}</p>
```

Comment:
Well done, but i'm not sure is here good practice to use nested ternary operators

Reply:
Good remark, please move ${sex === 'm' ? 'wife' : 'husband'} condition in separate variable